Formula/victoriametrics.rb

```diff
@@ -0,0 +1,69 @@
+class Victoriametrics < Formula
+  desc "Cost-effective and scalable monitoring solution and time series database"
+  homepage "https://victoriametrics.com/"
+  url "https://github.com/VictoriaMetrics/VictoriaMetrics/archive/v1.88.1.tar.gz"
+  sha256 "92803ae61f927b7173e25d4537c429bf3054edfa2f719bafc8fb9fcd411cab87"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    ENV.deparallelize
+
+    system "make", "victoria-metrics"
+    bin.install "bin/victoria-metrics"
+
+    (etc/"scrape.yml").write <<~EOS
+      global:
+        scrape_interval: 10s
+
+      scrape_configs:
+        - job_name: "victoriametrics"
+          static_configs:
+          - targets: ["127.0.0.1:8428"]
+    EOS
+    etc.install "scrape.yml"
+  end
+
+  service do
+    run [opt_bin/"victoriametrics_brew_services",
+      "--promscrape.config=#{etc}/scrape.yml",
+      "--storageDataPath=#{var}/victoriametrics-data",
+      "--retentionPeriod=12",
```

Comment:
I don't think we should set this

Reply:
Could you please share your example of configuration what will be ok for brew ? Because I'm trying to make the same structure as it work for linux OS via our rpm packages but don't understand the rules for brew and why it these config should be not the same as prometheus has?

Reply:
`--retentionPeriod=12` seems like something that you wouldn't want you package manager to set but rather a configuration that the user would want to set. Essentially I want brew to set as little as possible to be functional.

Reply:
Ok, removed it.